<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>869</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1120</width>
    <height>870</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_wc">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>780</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255);
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="iniciar_guia">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Guía</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salirv2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_traducir">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>770</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Traducir y Oír</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_inacap">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>759</y>
      <width>260</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Imagenes.qrc">:/Img/Inacap Logo.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_fase1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>510</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_fase2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>510</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Señas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tdapi">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>680</y>
      <width>781</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 16pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>Traduccíon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_borrar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>770</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_instru">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>770</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Instrucciones</string>
    </property>
    <property name="icon">
     <iconset resource="Imagenes.qrc">
      <normaloff>:/Img/pregunta.png</normaloff>:/Img/pregunta.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
